### PR TITLE
Add APIs to multipoolAutoswap for compatibility with Treasury

### PR DESCRIPTION
**Pull request by Chris-Hibbert**

### CODE REVIEW

File: packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js [hunk 1]
```diff
@@ -1841,3 +1841,32 @@ test('multipoolAutoSwap jig - insufficient', async t => {
...
+    installation,
+    harden({ Central: centralR.issuer }),
+    { timer: fakeTimer },
+  );
+  const aliceCollectFeesInvitation = await E(
+    creatorFacet,
+  ).makeCollectFeesInvitation();
+
+  const feeSeat = await zoe.offer(aliceCollectFeesInvitation);
+  t.deepEqual(await E(feeSeat).getCurrentAllocation(), {});
```
Comment (katelynsills) [inline packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js:1864]: This looks good! Can we test getting a payout, if that is an expected part of the fee extraction?

File: packages/zoe/src/contracts/multipoolAutoswap/multipoolAutoswap.js [hunk 2]
```diff
@@ -143,6 +143,16 @@ const start = zcf => {
     getPool,
   );
 
+  // compatibility with Treasury which expects to be able to collect fees from
+  // the AMM.
+  const makeCollectFeesInvitation = () => {
+    return zcf.makeInvitation(seat => seat.exit(), 'collect empty fee');
```
Comment (katelynsills) [inline packages/zoe/src/contracts/multipoolAutoswap/multipoolAutoswap.js]: For full compatibility, should we name the invitation the same thing? 
```suggestion
    return zcf.makeInvitation(seat => seat.exit(), 'collect Fees');
```

**Review by katelynsills — APPROVED** [review]
LGTM! One comment about a small additional test

**Review by katelynsills — DISMISSED** [review]
Revoked my previous review, because I think we need to add terms for the protocolFee and poolFee, otherwise the treasury will fail to use this. Are there any other changes that newSwap made?

File: packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/zoe/test/unitTests/contracts/test-multipoolAutoswap.js:1864]: added.

File: packages/zoe/src/contracts/multipoolAutoswap/multipoolAutoswap.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/zoe/src/contracts/multipoolAutoswap/multipoolAutoswap.js]: It's easy enough to do, though I don't see any way that it would help. 

Done

**Comment by Chris-Hibbert** [comment]
I've made the requested changes.

**Review by katelynsills — APPROVED** [review]
This code looks good to me. 

It looks like in collectRewardFees.js, we use `offerTo` to collect the fees. I think that will be ok with an empty object in place of the paymentKeywordRecord. Do you think that's the case? Do we need to test that somehow?

**Comment by katelynsills** [comment]
Dean's manual testing is good enough for me not to worry about `offerTo`

**Pull request merged**